modules/tinymce/CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The dialog API `setData` method now uses a deep merge algorithm to support partial nested objects #TINY-8333
 - The dialog spec `initialData` type is now `Partial<T>` to match the underlying implementation details #TINY-8334
 - Improved support for placing the caret before or after noneditable elements within the editor #TINY-8169
+- Notifications no longer require a timeout to disable the close button #TINY-6679
```

Comment:
Shouldn't it mention that moveRel and moveTo was removed as well?

Reply:
I don't believe they do, because they were never documented. They only exist in the types not the documented API.